Source/Eto.Gtk/Forms/Controls/SplitterHandler.cs

```diff
@@ -11,37 +11,126 @@ public class SplitterHandler : GtkContainer<Gtk.Paned, Splitter, Splitter.ICallb
 		SplitterOrientation orientation;
 		SplitterFixedPanel fixedPanel;
 		int? position;
+		double relative = double.NaN;
+		int suppressSplitterMoved;
+
+		private int _prefer(int width1, int width2)
+		{
+			if (position.HasValue)
+				width1 = position.Value;
```

Comment:
This could be nicely reused in WinForms as well, but that would need some common helper, or shared source. What do you think? Struct or class? Class is easier to use, struct has better footprint but using it is a bit harder (some methods need to be static with ref Helper me). I'd prefer class - struct would be overkill.